### ISSUE TITLE
HTTP request failed! HTTP/1.1 426 Upgrade Required

### ISSUE THREAD

**Issue by tretos53** [mattab/trello-backup#55]
Hi,

When running: `sudo /usr/bin/php /backups/trello/trello-backup.php` I get an error:

```
PHP Warning:  file_get_contents(https://api.trello.com/1/members/me/boards?&key=xxxxxx&token=xxxxxx): failed to open stream: HTTP request failed! HTTP/1.1 426 Upgrade Required
 in /backups/trello/trello-backup.php on line 53
Error requesting boards - maybe try again later and/or check your internet connection
```

PHP version:

```
$ sudo /usr/bin/php -v
PHP 7.0.33-29+ubuntu18.04.1+deb.sury.org+1 (cli) (built: May 14 2020 09:09:15) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.0.33-29+ubuntu18.04.1+deb.sury.org+1, Copyright (c) 1999-2017, by Zend Technologies
```

This script used to work fine but I haven't run it for a while so maybe something does need an update. OS ?

**Comment by martijnengler** [comment]
You are probably using an older version of the script. This was fixed in a6a65a1 (see #52 )

**Comment by mattab** [comment]
Thanks @martijnengler for answering!
